docs/namespaced-broker-resources.md

```diff
@@ -0,0 +1,159 @@
+---
+title: Namespaced Broker Resources
+layout: docwithnav
+---
+
+# Namespaced Broker Resources
```

Comment:
This causes the title to be printed twice since the layout also prints the title.